src/Writers/HTMLWriter.jl

```diff
@@ -224,6 +224,7 @@ function asset_links(src::AbstractString, assets::Vector)
         local ext = splitext(each)[end]
         local url = relhref(src, each)
         local node =
+            ext == ".ico" ? link[:href  => url, :rel => "shortcut icon", :type => "image/x-icon"] :
```

Comment:
Just `icon` should do I think -- `shortcut` doesn't exists in HTML5 as far as I know.

Reply:
I can try without it. it does work though

Reply:
Yup, HTML5 actually allows it as well, to be backwards compatible. But new HTML code should not use it, as far as I know.